.github/workflows/deploy-preview-cleanup.yml

```
@@ -94,48 +94,61 @@ jobs:
-          GH_TOKEN: ${{ secrets.DELETE_DEPLOYMENT_PAT_TOKEN }}   # must have delete-packages scope
+          GH_TOKEN: ${{ secrets.DELETE_DEPLOYMENT_PAT_TOKEN }}
           PKG_NAME: ${{ steps.vars.outputs.PACKAGE_NAME_PREVIEW }}
           PR_NUMBER: ${{ steps.vars.outputs.PR_NUMBER }}
           OWNER: ${{ github.repository_owner }}
         shell: bash
         run: |
           set -euo pipefail
 
-          echo "Looking for versions of '${PKG_NAME}' whose tags belong to PR #${PR_NUMBER} …"
+          HARD_CODED_PATH="/users/suddenlygiovanni/packages/container/suddenlygiovanni.dev%2Fsuddenlygiovanni.dev-preview/versions?per_page=100"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)